src/main/java/com/thebois/models/beings/actions/IActionGenerator.java

```diff
@@ -0,0 +1,23 @@
+package com.thebois.models.beings.actions;
+
+import com.thebois.models.beings.IActionPerformer;
+
+/**
+ * Represents a generator of actions.
```

Comment:
I mean, its the name of the interface :P

Reply:
Yeah, but that's all it is, both in name and in function. Sometimes the name is different than the function, like for `Being`, where the function is that it's an entity that can do things in the world. For this, though, which is actually just a very simple representation of a lambda method, I chose to name it exactly after its function, so they line up.

Couldn't come up with anything more fitting, maybe you can?

Any suggestions for another name, or another way to describe its functionality?

Reply:
Yeah that it better, and more consistent with the rest of the project 👍

Reply:
Fixing, but I don't see how it's more consistent? Do we have other `IxxxxSource` interfaces?